Lib/defcon/objects/base.py

```diff
@@ -445,6 +445,23 @@ def _serialize(self, getters, whitelist=None, blacklist=None, **kwargs):
             data[key] = getter(key)
         return data
 
+    # =============================================
+    # = ufo lib writer/reader validation settings =
+    # =============================================
+
+    ufoLibReadValidation = False
+    ufoLibWriteValidation = True
+
+    def _get_ufoLibReadValidate(self):
+        return self.ufoLibReadValidation
+
+    ufoLibReadValidate = property(_get_ufoLibReadValidate, doc="A bool indicating if the UFOReader should validate data while reading.")
```

Comment:
what do these `@property` getters/setters do more than the simple attributes? The property docstring?

Reply:
the style or properties in defcon is like this (not using a decorator)

Reply:
no, I didn't mean you should use `@property` decordator, I meant that the use of properties here is kind of redundant, since it doesn't do anything more that the normal instance attribute, only adds an extra method call. Anyway doesn't really matter, it's not that this is going to be called 1000000 times in a hot loop

Reply:
I see.. will change that

Reply:
changed